client/my-sites/post-relative-time-status/index.jsx

```diff
@@ -102,13 +107,18 @@ module.exports = React.createClass( {
 			statusText = this.getStatusText(),
 			realtiveTimeClass = ( timeText ) ? 'post-relative-time-status' : null,
 			innerText = ( <span>{ timeText }{ statusText }</span> ),
+			time = this.getTimestamp(),
 			details;
 
 		if ( this.props.link ) {
```

Comment:
Aside: To make refactoring like this easier, would be nice to apply necessary wrapping to `innerText`, eliminating `details` altogether:

```diff
diff --git a/client/my-sites/post-relative-time-status/index.jsx b/client/my-sites/post-relative-time-status/index.jsx
index 74c8c7fcb2..45879e7b02 100644
--- a/client/my-sites/post-relative-time-status/index.jsx
+++ b/client/my-sites/post-relative-time-status/index.jsx
@@ -107,20 +107,19 @@ module.exports = React.createClass( {
                        statusText = this.getStatusText(),
                        realtiveTimeClass = ( timeText ) ? 'post-relative-time-status' : null,
                        innerText = ( <span>{ timeText }{ statusText }</span> ),
-                       time = this.getTimestamp(),
-                       details;
+                       time = this.getTimestamp();
 
                if ( this.props.link ) {
                        const rel = this.props.target === '_blank' ? 'noopener noreferrer' : null;
-                       details = (
-                               <p className={ realtiveTimeClass } title={ time }>
-                                       <a href={ this.props.link } target={ this.props.target } rel={ rel } onClick={ this.props.onClick }>{ innerText }</a>
-                               </p>
+                       innerText = (
+                               <a href={ this.props.link } target={ this.props.target } rel={ rel } onClick={ this.props.onClick }>{ innerText }</a>
                        );
-               } else {
-                       details = ( <p className={ realtiveTimeClass } title={ time }>{ innerText }</p> );
                }
 
-               return details;
+               return (
+                       <p className={ realtiveTimeClass } title={ time }>
+                               { innerText }
+                       </p>
+               );
        }
 } );
```